migration-hev_mai_2020.sql

```diff
@@ -0,0 +1,9 @@
+SET names 'utf8mb4';
+SET character_set_database = 'utf8mb4';
+SET collation_database = 'utf8mb4_unicode_ci';
+SET CHARACTER SET utf8mb4, CHARACTER_SET_CONNECTION = utf8mb4;
+SET collation_connection = 'utf8mb4_unicode_ci';
```

Comment:
Is this really necessary?  The SQL dump should be exporting with these defaults. Is that not the case?

Reply:
> Is this really necessary? The SQL dump should be exporting with these defaults. Is that not the case?

This portion of code was added when I rebase with master, I haven't added it personally